src/canopy_cli/cli.py

```diff
@@ -260,8 +263,8 @@ def _batch_documents_by_chunks(chunker: Chunker,
         \b
         Upload local data files to the Canopy service.
 
-        Load all the documents from data file or a directory containing multiple data files.
-        The allowed formats are .jsonl and .parquet.
+        Load all the documents from a data file or a directory containing multiple data
+        files. The allowed formats are .jsonl and .parquet.
```

Comment:
Can you please add also .txt ? wee added support and probably forgot to document it here